src/views/Auth/Auth.jsx

```diff
@@ -9,13 +9,23 @@ import styles from './Auth.css';
 export default function Auth({ isSigningUp = false }) {
   const history = useHistory();
   const { setUser } = useUser();
-
+  
   const handleSubmit = async (email, password) => {
+    
     try {
       // TODO: Implement sign up & sign
       // Use isSigningUp to determine whether to sign up or sign in a user
-      // If signing in: set the user ({id, email}) and redirect to /notes
-      // If signing up: redirect to /confirm-email
+      if (isSigningUp){
```

Comment:
great logic here to check to see if the user is logged in or not to determine where to send the user